std_msgs/Header header
colorized_mesh_display/ColorizedMesh mesh
